src/main/java/brandkon/brand/entity/BrandCategory.java

```diff
@@ -0,0 +1,20 @@
+package brandkon.brand.entity;
+
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Data
+@Entity
+public class BrandCategory {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private Brand brand;
+    private Long CategoryId;
```

Comment:
👍 간접 참조로 끊으셨네요

소문자
```suggestion
    private Long categoryId;
```